docs/modules/customize/pages/social_shares.adoc

```diff
@@ -0,0 +1,62 @@
+= Social shares services
+
+Starting with the v0.27.0, you have the following social shares services that you can enable.
+
+== Configuration
+
+By using the `DECIDIM_SOCIAL_SHARE_SERVICES` environment variable, you can enable snervices:
```

Comment:
Looking out with a set fresh of eyes, I found out that I missed documenting this new environment variable. Can you add it please? 

It'd be docs/modules/configure/pages/environment_variables.adoc, I think in the ".Additional Optional Decidim Env Vars for the setting up the application" section.
```diff
diff --git a/docs/modules/configure/pages/environment_variables.adoc b/docs/modules/configure/pages/environment_variables.adoc
index b7829a56e2..337d8ee64c 100644
--- a/docs/modules/configure/pages/environment_variables.adoc
+++ b/docs/modules/configure/pages/environment_variables.adoc
@@ -639,6 +639,13 @@ Separate each item of the array with a comma AND a space, for instance:
 |
 |No
 
+|DECIDIM_SOCIAL_SHARE_SERVICES
+|Which Social Networks will be available when using the "Share" feature on some resources.
+
+More details at xref:customize:social_shares.adoc[Customize Social Shares documentation]
+|Twitter, Facebook, WhatsApp, Telegram
+|No
+
 |DECIDIM_ALLOW_OPEN_REDIRECTS
 |Disable the redirection to the external host when performing redirect back.

```

Reply:
Done. Did some small formatting changes to match with the other sections of that doc.

Moreover, I have also done some minor proofreading and formatting changes to the actual documentation page.